src/NSwag.Sample.NetCoreAngular/Program.cs

```diff
@@ -7,14 +7,15 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
```

Comment:
@RSuter \ @rynowak, the command requires that the app is structured in a way to allow getting to the `IWebHost`. The code path's pretty similar to how TestServer \ dotnet ef is set up and I've filed a work item (https://github.com/aspnet/Hosting/issues/1255) to make our internal package that these two use publicly available. 

Mvc's newly minted test package chooses to proceed with an empty WebHostBuilder (https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs#L123), when the webhost cannot be construed. This ends up being problematic since the service collection does not have nearly enough services to do meaningful work. In the aspnetcore2swagger command, I chose to throw pointing users to the 2.0 docs - https://docs.microsoft.com/en-us/aspnet/core/fundamentals/hosting?tabs=aspnetcore2x to use as a template.